### ISSUE TITLE
Impossible to implement removing in tce

### ISSUE THREAD

**Issue by anton2007** [willeccles/cpm#29]
Hi willecles! I want to contribute this package manager and add tce support, but tce require more than one command to remove extenstion.

1) remove it from /tce/onboot.list
2) reboot
3) remove extenstion from /tce/optional

**Comment by willeccles** [comment]
Yeah, I don't know if that is going to be possible. However, you could add everything else and then add a message for removal. See CONTRIBUTING.md.
